cmd/iam-store.go

```diff
@@ -1184,6 +1184,18 @@ func (store *IAMStoreSys) GetUsers() map[string]madmin.UserInfo {
 	return result
 }
 
+// GetUsersWithMappedPolicies - safely returns the name of access keys with associated policies
+func (store *IAMStoreSys) GetUsersWithMappedPolicies() map[string]string {
+	cache := store.rlock()
+	defer store.runlock()
+
+	result := make(map[string]string)
+	for k, v := range cache.iamUserPolicyMap {
```

Comment:
This map only contains users with directly mapped policies.

To get policies inherited from group memberships, see `cache.iamUserGroupMemberships` for a list of groups that the user belongs to and then lookup `cache.iamGroupPolicyMap` for a list of policies mapped to each such group.

Reply:
Actually this is not right for ldap. Need to check group membership in the credentials, object and then check mapped policies.